### DISCUSSION TITLE
DayZ Experimental fails to start

### DISCUSSION THREAD

**Discussion by cdnschr** [GameServerManagers/LinuxGSM#4116] [Q&A]
Installation was successful, however after starting it crashes with a segfault error:
11:07:57 Mission read.
11:07:57 Connected to Steam
11:07:57 Steam policy response
Received fatal signal: SIGSEGV(11) PID: 16970
***** SIGNAL SIGSEGV (11)
exitWithDefaultSignalHandler:156. Exiting due to signal, 11

Using Ubuntu 22.04 VM with 4 Cores and 8GB RAM.

**Comment by TexmanDIXX** [comment]
I'm having same issue. DayZServer is the only one I have a problem with. I also have 7D2D, Rust and Ark servers that run on this machine. I only run one at a time though. It used to run without a problem but after an update it started this. I've also deleted the user and home directory and started from scratch with same results. Don't know what direction to go at this point

Ubuntu 22.04 64bit / Linux 5.15.0-67-generic
HP Z440
Intel Xeon E5-1650 v3 6c 12t
80GB Ram
1TB SSD hosts OS

Received fatal signal: SIGSEGV(11)      PID: 138482
***** SIGNAL SIGSEGV (11)
exitWithDefaultSignalHandler:156. Exiting due to signal, 11

[dzserve.log](https://github.com/GameServerManagers/LinuxGSM/files/10930975/dzserve.log)

**Comment by KraftwerkBeard** [comment]
Same exact thing for me. Started with a fresh 22.04 server and made no changes to lgsm default configs. Gives the same error as above. grabbed the console output for some context.

 6:56:29 [46] VehicleTruck01
 6:56:29   Primary Spawner: "Vehicle", active: yes, repeat: 1
 6:56:29   Primary Spawner: "Vehicle", active: yes, repeat: 1
 6:56:29 [CE][Hive] :: Initializing of spawners done.
 6:56:29 [CE][Hive] :: Initializing of spawners done.
 6:56:29 [CE][Hive] :: Init sequence finished.
 6:56:29 [CE][Hive] :: Init sequence finished.
 6:56:29 Player connect enabled
 6:56:29 Player connect enabled
 6:56:29 Mission read.
 6:56:29 Connected to Steam
 6:56:29 Steam policy response
Received fatal signal: SIGSEGV(11)      PID: 31754
***** SIGNAL SIGSEGV (11)
exitWithDefaultSignalHandler:156. Exiting due to signal, 11
/home/dayzserver-v/lgsm/functions/command_debug.sh: line 121: 31754 Segmentation fault      (core dumped) ./DayZServer -ip=0.0.0.0 -port=2302 -config=/home/dayzserver-v/serverfiles/cfg/dayzserver.server.cfg -mod= -servermod= -bepath= -limitFPS=60 -dologs -adminlog -freezeCheck
[ ERROR ] Debuging dayzserver: Server has stopped: exit code: 0
[ ERROR ] Debuging dayzserver: Press ENTER to exit debug mode

I assume this is a base issue with this version of Linux and the DayZ server. Probably not a linuxGSM thing, but in hopes they look into this at any point. I am trying to figure this out on the DayZ side as well. Need to get my server on an older version of Ubuntu I suppose.

**Comment by DraganKov** [comment]
Running on Mint, first time I've had this issue come up for me with any game server.  I ran it with 0 changes then tried the below command to eliminate the possibility of a similar problem as Arma 3 but yeah, here's my log. 

22:30:39 [46] VehicleTruck01
22:30:39   Primary Spawner: "Vehicle", active: yes, repeat: 1
22:30:39 [CE][Hive] :: Initializing of spawners done.
22:30:39 [CE][Hive] :: Init sequence finished.
22:30:39 Player connect enabled
22:30:39 Mission read.
22:30:39 Connected to Steam
22:30:39 Steam policy response
/home/dayzserver/lgsm/functions/command_debug.sh: line 121: 2621708 Segmentation fault      (core dumped) ./DayZServer -ip=0.0.0.0 -port=2302 -config=/home/dayzserver/serverfiles/cfg/dayzserver.server.cfg -bepath= -limitFPS=60 -freezeCheck
[ ERROR ] Debuging dayzserver: Server has stopped: exit code: 0
[ ERROR ] Debuging dayzserver: Press ENTER to exit debug mode

**Comment by jetkai** [comment]
Issue seems to be caused by updated libraries on the OS (likely glibc) - However downgrading this library can cause a ton of issues.

Using an older OS like Ubuntu 18.04 / RHEL8 (or Windows), are the only feasible workaround I could find until this is resolved by the game devs.

[LINK TO ACTIVE BUG](https://feedback.bistudio.com/T170140) - has been under investigation for 3 months tho, unsure of when it'll be fixed.
